courses/1-ramda-basics/2-composition.js

```diff
@@ -110,7 +110,6 @@ describe("Course 1.2 - Composition", () => {
     it("Split names into an array, capitalize both first and last names, sort by last name", () => {
 
       const namePipeline = R.compose(
-        joinNames, sortByLastName, capitalize, splitNames
         // Provide the functions defined above in the correct order here
         joinNames,
```

Comment:
I see you went back to make it line-wise. This is definitely an improvement and should be easier to read and editor over time.